python/tidy/servo_tidy_tests/rust_tidy.rs

```diff
@@ -41,4 +41,8 @@ impl test {
             && x;
     }
 
+    fn test_fun3(x:&Root<isize>) {
+        ()
+    }
+
```

Comment:
Well, this won't be needing a new function, but well, it's just a test... :smile: